.circleci/src/jobs/bump_version.yml

```diff
@@ -18,4 +18,21 @@ steps:
   - run:
       name: Calculate next version code
       command: echo "export NEW_VERSION_CODE=$(app-toolbelt v0 version calc | jq .versionCode)" >> ${BASH_ENV}
+  - when:
+      condition: << parameters.prepare_delivery >>
+      steps:
+        - run:
+            name: Bump git version
+            command: app-toolbelt v0 release bump-to ${NEW_VERSION_NAME} ${NEW_VERSION_CODE} --deliverino-private-key ${DELIVERINO_PRIVATE_KEY} --owner ${CIRCLE_PROJECT_USERNAME} --repo ${CIRCLE_PROJECT_REPONAME} --branch ${CIRCLE_BRANCH}
+# TODO 1199
+#  - when:
+#      condition:
+#        and:
+#          - << parameters.prepare_delivery >>
+#          - not:
+#              equal: [main, << pipeline.git.branch >>]
```

Comment:
why not equal?

Reply:
This is the condition for the step afterwards, which notifies us on mattermost, we need to merge this branch before a new release, because otherwise the next release will fail due to the same version numbers.